src/cf-tables/src/TableRowLinks.js

```diff
@@ -1,38 +1,38 @@
 /* ==========================================================================
    Table Row Links
 
-   Mixin for adding row link click functionality to table organism.
-
+   Mixin for adding row link click functionality to Table organism.
    ========================================================================== */
 
 'use strict';
 
 const closest = require( 'cf-atomic-component/src/utilities/dom-closest' ).closest;
 
 const TableRowLinks = {
+  ui: {
+    base: '.o-table__row-links'
+  },
 
   events: {
     'click tbody tr': 'onRowLinkClick'
   },
 
-  ui: {
-    base: '.o-table__row-links'
-  },
+  onRowLinkClick: onRowLinkClick
+};
 
-  /**
-   * Handle a click of the table.
-   *
-   * @param {Object} event Mouse event for click on the table.
-   */
-  onRowLinkClick: function( event ) {
-    let target = event.target;
-    if ( target && target.tagName === 'A' ) {
-      return;
-    }
-    target = closest( event.target, 'tr' );
-    var link = target.querySelector( 'a' );
-    if ( link ) window.location = link.getAttribute( 'href' );
+/**
+ * Handle a click of the table.
+ *
+ * @param {MouseEvent} event - Mouse event for click on the table.
+ */
+function onRowLinkClick( event ) {
+  let target = event.target;
+  if ( target && target.tagName === 'A' ) {
+    return;
   }
-};
+  target = closest( event.target, 'tr' );
+  const link = target.querySelector( 'a' );
+  if ( link ) window.location = link.getAttribute( 'href' );
```

Comment:
This should be wrapped in curly braces to match the return on L:31

Reply:
Fixed